### PR TITLE
Updates for new log output. Minor consistency fixes

### PR DESCRIPTION
**Pull request by michael-w-williams**
Came up with description of log format from the Open Function format docs.

### CODE REVIEW

File: Async/README.md [hunk 1]
```diff
@@ -86,20 +86,26 @@ Status: success
...
 
 ```json
-{"message":"Hello World"}
+{"body":"{\"message\":\"Hello World\"}\n","content_type":"","protocol":{"status_code":200}}
 ```
 
+A JSON response is returned with three fields.
+
+* **body:** The output from the function. In this case an escaped version of the JSON response message: `{"message":"Hello World"}`
+* **content_type:** The content type of the response message. Empty in this case which defaults to `application/json`.
```
Comment (shaunsmith) [inline Async/README.md]: The  empty content type is a problem in the Go FDK.  The boilerplate code doesn't specify it so it should either not be in the response or default to something generic like application/text as we are doing when `fn call` is invoked with no content type specified. The fact that the FDK is automatically logging the result (with user data) to stderr is also a problem I'll look into. 

For now I'd comment out the content_type discussion until we can get it cleared up.

Comment (shaunsmith) [reply Async/README.md]: Hmm, might not be the FDK logging the result.  Might be the Fn server.

Comment (michael-w-williams) [reply Async/README.md]: I will just take out that bullet for now.

**Comment by michael-w-williams** [comment]
OK. Removed the content-type discussion.

**Comment by michael-w-williams** [comment]
@shaunsmith @sachin-pikle Made the changes we discussed on Tuesday. Should be ready to go.

**Comment by michael-w-williams** [comment]
This should be ready now @shaunsmith and @sachin-pikle.

**Pull request merged**